locales/en/yourloops.json

```diff
@@ -21,6 +21,7 @@
   "account-created-info-2-hcp": "Clicking the continue button will take you to your dashboard. You can then join or create care teams, and invite your patients so as to share their data.",
   "account-creation-finalization": "Account creation finalization",
   "account-creation-finalized": "Account creation finalized",
+  "account-flagged-for-deletion": "Your account has been flagged for deletion.",
```

Comment:
Same remark as for seashell ;)
The message should be more explicit and explain that the access is suspended/removed